manifests/init.pp

```diff
@@ -299,14 +313,17 @@
   Boolean $package_crmsh                                             = $corosync::params::package_crmsh,
   Boolean $package_pacemaker                                         = $corosync::params::package_pacemaker,
   Boolean $package_pcs                                               = $corosync::params::package_pcs,
+  Boolean $package_fence_agents                                      = $corosync::params::package_fence_agents,
   Optional[Array] $packageopts_corosync                              = $corosync::params::package_install_options,
   Optional[Array] $packageopts_pacemaker                             = $corosync::params::package_install_options,
   Optional[Array] $packageopts_crmsh                                 = $corosync::params::package_install_options,
   Optional[Array] $packageopts_pcs                                   = $corosync::params::package_install_options,
+  Optional[Array] $packageopts_fence_agents                          = $corosync::params::package_install_options,
   String $version_corosync                                           = $corosync::params::version_corosync,
   String $version_crmsh                                              = $corosync::params::version_crmsh,
   String $version_pacemaker                                          = $corosync::params::version_pacemaker,
   String $version_pcs                                                = $corosync::params::version_pcs,
+  String $version_fence_agents                                       = $corosync::params::version_fence_agents,
```

Comment:
I think an empty string isn't valid here, right? So we could improve the datatype:
```suggestion
  String[1] $version_fence_agents                                    = $corosync::params::version_fence_agents,
```